prow/scripts/cluster-integration/kyma-gke-integration.sh

```diff
@@ -220,7 +220,7 @@ fi
 if [ -z "${CLUSTER_VERSION}" ]; then
       export CLUSTER_VERSION="${DEFAULT_CLUSTER_VERSION}"
 fi
-CLEANUP_CLUSTER="true"
+CLEANUP_CLUSTER=""
```

Comment:
As it is "testing job" this change should be implemented in your private repository or as  a new version of the script, for now you are overwriting production script.

Reply:
Reverted CLEANUP_CLUSTER value to original.